frontend/src/app/commonComponents/SRToastContainer.test.tsx

```diff
@@ -9,6 +10,14 @@ const renderSRToastContainer = () => {
 };
 
 describe("SRToastContainer", () => {
+  beforeAll(() => {
+    MockDate.set(1667580636341);
```

Comment:
I am curious about the `MockDate` implementation here. Is this just a random hard-coded number? I see that it is used to identify the alert in the corresponding snap test.

Reply:
yeah, here is being used to make sure the unit testing gets a consistent value (instead of unique everytime) so the snapshots do not raise false positives.